python/res/fm/ecl/ecl_config.py

```diff
@@ -59,68 +71,134 @@ class EclConfig(object):
     example file.
 
     """
-    DEFAULT_CONFIG_FILE = os.path.join(os.path.dirname(__file__), "ecl_config.yml")
 
-    def __init__(self):
-        config_file = os.getenv("ECL_SITE_CONFIG", default = self.DEFAULT_CONFIG_FILE)
+    def __init__(self, config_file):
         with open(config_file) as f:
             try:
                 config = yaml.load(f)
             except:
                 raise ValueError("Failed parse: {} as yaml".format(config_file))
 
         self._config = config
+        self._config_file = os.path.abspath(config_file)
+
+
+    def __contains__(self, version):
+        if version in self._config[Keys.versions]:
+            return True
+
+        if self.default_version is None:
```

Comment:
`return self.default_version is not None and version in  [None, Keys.default]` to make the function a bit more compact?